docs/_docs/hivemq/data-policy.md

```diff
@@ -1,30 +1,30 @@
 ---
 nav_order: 1
-redirect_from: /docs/hivemq/policies.html
+redirect_from: /docs/hivemq/data-policy.html
```

Comment:
Maybe preserve the redirect of "/docs/hivemq/policies.html" in addition, so that existing links do not break.